keps/sig-storage/1855-csi-driver-service-account-token/README.md

```diff
@@ -173,14 +173,19 @@ Option 1 is adopted. See discussion
 
 ### Graduation Criteria
 
-#### Alpha->Beta
+#### Alpha
 
 - Implemented the feature.
 - Wrote all the unit and E2E tests.
 
+#### Alpha->Beta
+
+- Deployed the feature in production and went through at least minor k8s.
+- Fixed any bugs.
+
 #### Beta->GA
 
-- Deployed the feature in production and went through at least minor k8s
+- Deployed the feature in production and went through at least minor k8s.
```

Comment:
Do you want to add some stress/scale testing to make sure we can handle a lot of pods using this feature?

Reply:
done